patchwork-user-profiles.md

```diff
@@ -1,299 +0,0 @@
-# Patchwork: User profiles
```

Comment:
nah, but i understand ssbc/docs to be about documentation relating to the project, not just the api docs, so it could fit in somewhere.